sysdeps/atlas/include/abi-bits/seek-whence.h

```diff
@@ -0,0 +1,8 @@
+#ifndef _ABIBITS_SEEK_WHENCE_H
+#define _ABIBITS_SEEK_WHENCE_H
+
+#define SEEK_SET 0
+#define SEEK_CUR 1
+#define SEEK_END 2
+
+#endif // _ABIBITS_SEEK_WHENCE_H
```

Comment:
(using this file as an example, this applies to all sysdeps)

Installed headers cannot have line comments in them, as these aren't usable on some C compilers and versions.
You should probably use existing abi-bits with symlinks instead of specifying an entirely new ABI.  See any other port for an example.